spec/integration/yardstick/document/file_spec.rb

```diff
@@ -12,5 +12,5 @@
 
   it { should be_kind_of(Pathname) }
 
-  it { should == file }
+  it { should eq(file) }
```

Comment:
I think we should use the most "strongest" matcher to specify the expectation. In that case we should use `should eql(file)`.

`eq` is calling `#==` internally, and `#==` is less "stronger" (contains more semantics) than `#eql?` which is the backend for the `eql` expectation.